stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/reaction/internal/ReactionDao.kt

```diff
@@ -28,8 +28,23 @@ internal interface ReactionDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     suspend fun insert(reactionEntity: ReactionEntity)
 
-    @Query("SELECT * FROM stream_chat_reaction WHERE stream_chat_reaction.syncStatus IN (:syncStatus)")
-    suspend fun selectSyncStatus(syncStatus: SyncStatus): List<ReactionEntity>
+    @Query("SELECT * FROM stream_chat_reaction WHERE id = :id")
+    suspend fun selectReactionById(id: Int): ReactionEntity?
+
+    @Query("SELECT * FROM stream_chat_reaction WHERE id IN (:ids)")
+    suspend fun selectReactionsByIds(ids: List<Int>): List<ReactionEntity>
+
+    @Query(
+        "SELECT id FROM stream_chat_reaction " +
+            "WHERE syncStatus = :syncStatus ORDER BY syncStatus ASC LIMIT :limit"
```

Comment:
Follow the pattern of your PR, I believe you should break the line before the ORDER BY

Reply:
good catch 👍